includes/abstracts/abstract-wc-gateway-ppec.php

```diff
@@ -559,6 +559,11 @@ public function generate_image_html( $key, $data ) {
 		// For backwards compatibility (customers that already have set a url)
 		$value_is_url = filter_var( $value, FILTER_VALIDATE_URL ) !== false;
 
+		if ( function_exists( 'add_image_size' ) ) {
```

Comment:
To answer one of your questions, while this seems to work here, I think the constructor would be a better place for these `add_image_size()` calls.

Reply:
Done. Tested. Works. ✅